### ISSUE TITLE
Better groupping experience

### ISSUE THREAD

**Issue by Ursanon** [openvinotoolkit/cvat#3013]
### My actions before raising this issue
- [x] Read/searched [the docs](https://github.com/opencv/cvat/tree/master#documentation)
- [x] Searched [past issues](/issues)

### Context
Groupping task is very difficult currently.
We have no information about groups on side panel and colors between groups can be too similar. 

![image](https://user-images.githubusercontent.com/8686134/112447446-9f55cd80-8d51-11eb-9523-d1e29d5a5d4d.png)
![image](https://user-images.githubusercontent.com/8686134/112447696-e643c300-8d51-11eb-9efe-607fdc527617.png)
On side panel we have only instance ID without any information about group (other than color).

Filtering items by group and showing group ID should be very helpful.

To validate: we could iterate through all groups and show each individually

**Comment by Ursanon** [comment]
I think this feature will be a big step forward in the groupping feature.
cc @nmanovic

**Comment by nmanovic** [comment]
@Ursanon , I agree that it can be very helpful. Could you please help us to implement the feature?
